packages/subgraph/tasks/deploy.sh

```diff
@@ -126,7 +126,11 @@ deploy_to_goldsky() {
     local network="$1"
     # TODO: use tagging?
 
-    local subgraphName="protocol-$DEPLOYMENT_ENV-$network/1.0.0"
+    #Get subgraph version from package.json
+    PACKAGE_JSON_PATH="package.json"
+    VERSION_LABEL=$($JQ -r '.version' $PACKAGE_JSON_PATH)
```

Comment:
It's slightly confusing that`VERSION_LABEL` is an input but gets overriden here; but yeah, otherwise this is the version we want here.

Reply:
Changed it to SUBGRAPH_VERSION.